### PR TITLE
Initial wiring

### PR DESCRIPTION
**Pull request by rossf7**
Adds initial wiring based on what we do in devctl.

### CODE REVIEW

**Comment by rossf7** [comment]
OK should be good now. This adds the wiring with the `version` command. Next PR will be to add a `create cluster` command based on the `e2ekindprototype` code.

File: cmd/runner.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) Run(cmd *cobra.Command, args []string) {
```
Comment (xh3b4sd) [inline cmd/runner.go]: I think we can have some `RunE` or `RunError` interface to do the logging and process termination in fewer places.

File: main.go [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+
+	"github.com/giantswarm/e2ectl/cmd"
+)
+
+var (
+	gitCommit = "n/a"
+	source    = "https://github.com/giantswarm/e2ectl"
+)
+
+func main() {
```
[the remark this answers is not in the document]
Comment (rossf7) [reply main.go]: Good shout. Done.

File: cmd/runner.go (hunk 1 above)
Comment (rossf7) [reply cmd/runner.go]: That makes sense but where should it live? microkit?

Comment (xh3b4sd) [reply cmd/runner.go]: So when this is the method which is called by cobra then it is done there. Good question what happens on error then, but there is something like `RunE` and I thought we should rather use this and do normal error handling here.

Comment (rossf7) [reply cmd/runner.go]: Got it. Yes using `RunE` is nice and the error is passed to Execute when it's called. I renamed `Run` to `RunWithError` to align with mainWithError.

@xh3b4sd PTAL

**Review by xh3b4sd — APPROVED** [review]

**Pull request merged**